src/extra/theme.dark.css

```diff
@@ -13,4 +13,19 @@
     --surface-3: var(--gray-7);
     --surface-4: var(--gray-6);
   }
+}
+
+[data-theme=dark] {
+  color-scheme: dark;
+
+    --link: var(--indigo-3);
+    --link-visited: var(--grape-3);
+
+    --text-1: var(--gray-1);
+    --text-2: var(--gray-4);
+
+    --surface-1: var(--gray-9);
+    --surface-2: var(--gray-8);
+    --surface-3: var(--gray-7);
+    --surface-4: var(--gray-6);
```

Comment:
i'm thinking of making this into a file, letting someone import it like `@import "open-props/theme-switch/dark";`. These other spots with dark preference media queries also need duplicated though too, i wonder if they should be in the same file? 🤔

Reply:
This is definitely a great plan. I'm mildly concerned with the dark preference duplication being hard to manage across multiple files which is why I started with including it in all situations.

If _every_ use of `@media (--OSDark)` or `@media (--OSLight)` was in the theme file I'd agree with you 100%.

Reply:
You could *pretty* easily write an automatic refactor-er (real word, promise!) that would split the media queries for OSDark and OSLight into two and send them into two different files.

Reply:
def agree there's ways to automate this AND agree it's not fun to manage the media queries + attributes. there's a css proposal that would help here a lot called [`:media()`](https://github.com/w3c/csswg-drafts/issues/6247) that would allow combining the attribute selector and the media query into 1 selector, but it's only a proposal.